### ISSUE TITLE
Don't use `ToEventStream` as inputs

### ISSUE THREAD

**Issue by CJ-Wright** [xpdAcq/SHED#152]
Currently when we walk up the graph we then walk down to check if the node has a `ToEventStream` subscribed to the node. If so we use that `ToEventStream` as our input uids allowing us to only cache that part of the pipeline. However this assumed that data which came out of a `ToEventStream` node was stored somewhere, which might not always be true. We could inspect if the `ToEventStream` node has a `db.insert` decedent, but that might be a bit brittle. Additionally this very funny walk might be more computationally expensive than calling `nx.ancestors` and looking for `FromEventStream` instances. (Note that this might be able to be mitigated by playing some fun graph games, maybe by searching the graph for all `FromEventStream` nodes and all `ToEventStream` nodes, looking at the paths to those nodes and finally making decisions based off the path (eg is the `ToEventStream` attached to a node on the path?)

tl;dr our magic for `ToEventStream` input tracking is nice, but maybe too smart for its own good. We might remove it to improve init time, simplicity, and developer sanity.
